.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Question
```

Comment:
Do we want to have questions inside the GH repo instead of bug reports & feature requests only?

Reply:
I think for modules it's good enough to start, they can always remove it or change it later on